### PR TITLE
Add email confirmation views

### PR DESCRIPTION
**Pull request by eugenialcaraz**
#### Trello board reference:

* [#4](Create Sign Up Page)

---

#### Description:

* Add email confirmation page
* Add confirmation done page, with sign in view
* Add react-hook-form library as recommended
* Refactor forms with the new library

---

#### Reviewers:

* @nicmalegre 
* @jpirazusta 
* @tobicorradi 
* @diego317 

---

#### Notes:

* The connection is not implemented, will be added in a separated ticket
* Only the pages and the navigation were implemented in this PR

---

#### Preview:

image

image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { Routes, Route } from "react-router-dom";
-import { SignIn, About } from "@components/index";
+import { SignIn, About, EmailConfirmation, NewUser } from "@components/index";
 import { Landing, Pages } from "./pages";
 
 import { urlFormat } from "./utils/urlFormat";
```
Comment (nicmalegre) [inline src/App.tsx]:
```suggestion
import { Landing, Pages } from "@/pages";

import { urlFormat } from "@/utils/urlFormat";
```

File: src/components/common/input/index.tsx [hunk 2]
```diff
@@ -1,16 +1,34 @@
...
-  </div>
-);
+const Input: FC<InputProps> = ({
+  label,
+  stylesName,
+  type = "text",
+  register,
+  required,
+}) => {
+  return (
```
Comment (jpirazusta) [inline src/components/common/input/index.tsx]:
```suggestion
}) => (
```

File: src/components/landing/newUser/NewUser.module.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  font-size: 1.8rem;
+  line-height: 2.1rem;
+  font-weight: 600;
+  margin-top: 1.3rem;
+}
+.container > span:nth-child(2) {
+  font-weight: 400;
+  margin-top: 2.2rem;
+}
+.container > span:nth-child(3) {
```
Comment (jpirazusta) [inline src/components/landing/newUser/NewUser.module.css]:
```suggestion
}

.container > span:nth-child(2) {
  font-weight: 400;
  margin-top: 2.2rem;
}

.container > span:nth-child(3) {
```

**Review by nicmalegre — APPROVED** [review]

**Review by jpirazusta — APPROVED** [review]
👏

**Pull request merged**